templates/waggledance.json

```diff
@@ -34,6 +34,12 @@
        "name": "BASTION_SSH_KEY_ARN",
        "value": "${bastion_ssh_key_arn}"
      }
-    ]
+   ],
+   "healthCheck": {
+       "command": ["CMD-SHELL", "curl -f http://localhost:18000/actuator/health || exit 1"],
+       "interval": 5,
```

Comment:
Just checking that this is measured in seconds and not milliseconds ;)

Reply:
What happens when that URL is working but returning failure code?
I assume it only exist with code 1 when service is not responding on that port.

Ideally we should configure route 53 service discovery health check and check for http return codes.
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-discovery.html

Reply:
The "-f" flag handles that part, from: https://curl.haxx.se/docs/manpage.html

> -f, --fail

> (HTTP) Fail silently (no output at all) on server errors. This is mostly done to better enable scripts etc to better deal with failed attempts. In normal cases when an HTTP server fails to deliver a document, it returns an HTML document stating so (which often also describes why and more). This flag will prevent curl from outputting that and return error 22.

> This method is not fail-safe and there are occasions where non-successful response codes will slip through, especially when authentication is involved (response codes 401 and 407).